docs/getting_started/client.md

```diff
@@ -0,0 +1,258 @@
+# Using the OPTIMADE client
+
+This package includes a Python client that can be used to query multiple OPTIMADE APIs simultaneously.
+The client can be used from the command-line (`optimade-get`), or called in Python code.
+
+The client requires some extra dependencies that can be installed with the PyPI package with
+
+```shell
+pip install optimade[http_client]
+```
+or from a local copy of the repository with
+```shell
+pip install -e .[http_client]
+```
+
+By default, the client will query all OPTIMADE API URLs that it can find via the [Providers list](https://providers.optimade.org):
+
+
+=== "Command line"
+    ```shell
+    optimade-get --filter 'elements HAS "Ag"'
```

Comment:
Yeah, I'm not sure what the best default is really. For the CLI, it might make sense to have a default output file name that it writes to (similar to how curl works for files).

Currently it will limit the results to 10 per provider by default at the CLI, which can be increased with `max_results_per_provider` (I should add a docs example for this), so it should not be wasted.

The idea is that the output can be piped to another tool or redirected to a file though, following the unix philosophy (again I should add an example).

Reply:
Another option I considered is making the default CLI behavior "count" and adding a new flag for downloading the results (`optimade-get --get` feels a bit cumbersome though)

Reply:
It seems that there is some delay on the page  It only just updated so I did not see your comments before my last post.

Reply:
In most cases, you also want to have a look at what kind of structures you are going to get, so sending 10 structures along seems reasonable. So I do not think we have to make count the default. I just do not want to put too much load on the OPTIMADE servers with these example queries. When you do this in a tutorial like setting many people will execute these commands at the same time.

Reply:
> It seems that there is some delay on the page It only just updated so I did not see your comments before my last post.

Are you using the normal website or a desktop client? I imagine most of the issues are caused by force pushes that rewrite the history of the PR (hence why some commits look more recent).

Reply:
> Are you using the normal website or a desktop client? I imagine most of the issues are caused by force pushes that rewrite the history of the PR (hence why some commits look more recent).

I am using the website.
Force pushes can indeed break things, so it would be better to try to avoid using them.

Reply:
Unfortunately they are a necessary evil when rebasing, I think the only issue is that comments end up hidden above the commits they apply to, so you have to go looking for them a bit. You can also still see them in the files changed tab (but not for a particular commit).

Reply:
In usually merge changes from the master branch into my branch. 
And than do a squash merge when the PR is accepted.
Is there a reason I should do a rebase instead?